### DISCUSSION TITLE
How Sidebery is going to be affected with new Firefox native Vertical tabs?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [mbnuqw/sidebery#1760] [General]
> We’re calling on our community to test the new sidebar and vertical tabs experience and your constructive feedback is crucial as we refine these features. Please share your thoughts [on Mozilla Connect](https://connect.mozilla.org/t5/discussions/firefox-sidebar-and-vertical-tabs-try-them-out-in-nightly/m-p/63231#M22243) – we take your input into account and it helps us create a browsing experience that meets your needs 🦊.

> If you’re a web extension developer and your extension uses sidebar APIs or works with tabs, we’d love you to test it with the new sidebar and vertical tabs enabled. While there are no changes to Web Extension APIs tied to these features, this is a good opportunity to anticipate unforeseen issues resulting from the updated UI.

https://blog.nightly.mozilla.org/2024/08/07/firefox-sidebar-and-vertical-tabs-try-them-out-in-nightly-firefox-labs-131/

**Comment by llc0930** [comment]
Unless there is an official intention to change the practice of only backing up all open tab states to a single archive, I prefer Sidebery because of its additional snapshot functionality.
I have lost thousands of tabs many times after unexpected power outages...very painful...

**Reply by KAGEYAM4** [reply]
Yes sidebery is awesome, native-vertical tabs not gonna replace it for me ever.

I wanted to know, as vertical-tabs are going to be now natively supported, maybe sidebery can leverage more new functionalitly like animations and other things which might not be possible before ( i am not sure ).  And as mozilla is looking for feedback on this, maybe sidebery can participate in this.

**Comment by llc0930** [comment]
The official article also mentioned at the end: "While there are no changes to Web Extension APIs tied to these features, this is a good opportunity to anticipate unforeseen issues resulting from the updated UI."
It seems like there is no new API available?

On the contrary, I think the implementation of [this API](https://github.com/w3c/webextensions/issues/231) is more worth looking forward to.
I'm under the impression that Sidebery has already implemented this functionality somewhere for use.

**Reply by KAGEYAM4** [reply]
Ohk, so sidebery doesn't have to do anything on its side. On the other hand improvements for UI/UX going to be made to Sidebar from firefox as part of Vertical tabs native support.

**Reply by llc0930** [reply]
At least that's how I feel. I don't know what the author thinks lol.

**Comment by albino1** [comment]
Does the Firefox version support trees and grouping?

**Reply by llc0930** [reply]
At this stage the answer seems to be no?

**Reply by albino1** [reply]
> At this stage the answer seems to be no?

That makes it kind of useless IMHO.  Even Edge allows you to group.  So it doesn't sound like Sidebery or TST should be going anywhere anytime soon.

**Comment by thoughtsunificator** [comment]
At the very least, they seem to be going in the right direction.

**Comment by AnonJohn** [comment]
We all know Sidebery has a rich feature set that will surpass native vertical tabs for the foreseeable future. So let me ask a variant: What would it take for sidebery to BE the new vertical tabs? Are the licenses compatible? Could Mozilla be convinced (and thus save their time and resources at a time when they need them)?

The first question is: Would @mbnuqw  want this?
